paper.md

```diff
@@ -35,15 +35,15 @@ This is an early example of a *permutation test*. Like *Monte Carlo tests* [@rob
 
 Scientists and engineers frequently analyze data by considering questions of the following forms.
 
-1. Are the observations sampled from a hypothesized distribution? For instance, in Darwin's experiment, are the differences in heights between paired plants normally distributed?
+1. Are the observations sampled from a hypothesized distribution? For instance, in Darwin's experiment [@darwin1877effects], are the differences in heights between paired plants normally distributed?
 2. Are the samples drawn from the *same* distribution? For example, does the fertilization method have no effect on plant height?
 3. What can be inferred from the samples about the true value of a population statistic? For instance, what is the uncertainty in the true mean difference in heights between paired plants?
 
 Monte Carlo hypothesis tests, permutation tests, and the bootstrap are general procedures for answering questions of these forms, respectively.
 
 Although there are a variety of specialized statistical procedures for answering such questions (e.g., normality tests, t-tests, standard error of the mean), most computer implementations provide answers that are only guaranteed to be accurate when assumptions that simplify the mathematics are met (e.g., the sample size is large, there are no ties in the data). At the expense of additional computation, the `scipy.stats` functions `monte_carlo_test`, `permutation_test`, and `bootstrap` provide accurate answers to questions 1-3 even when the simplifying assumptions of specialized procedures are not met. Furthermore, these functions can be used to answer questions 1-3 even when specialized procedures have not been *developed*, giving users tremendous flexibility in the analysis that they can perform.
 
-Before the release of SciPy 1.9.0, the need for these procedures was partially met in the scientific Python ecosystem by tutorials (e.g., blog posts, medium.com) and niche packages, but the functions in SciPy have several advantages:
+Before the release of SciPy 1.9.0, the need for these procedures was partially met in the scientific Python ecosystem by tutorials (e.g., [@ye2020; @saenger2019; @omar2021]) and niche packages (e.g., [@resample2020; @permute2021]), but the functions in SciPy have several advantages:
```

Comment:
This renders as intended:

![image](https://user-images.githubusercontent.com/6570539/231860760-56c27fb2-6d24-408b-8672-240ee444860b.png)

Reply:
Yes, this is what I was thinking. Looks good @mdhaber!